db/seeds/04_game_join_companies.js

```diff
@@ -1,10 +1,24 @@
-const { generateGameJoinCompanies } = require('../utils');
+// const { generateGameJoinCompanies } = require('../utils');
+
+// exports.seed = function(knex) {
+//   // Deletes ALL existing entries
+//   return knex('game_join_companies').del()
+//     .then(() => {
+//       // Inserts seed entries
+//       return knex('game_join_companies').insert(generateGameJoinCompanies());
+//     });
+// };
+
+
+const path = require('path');
+const seedFile = require('knex-seed-file');
+
 
 exports.seed = function(knex) {
   // Deletes ALL existing entries
   return knex('game_join_companies').del()
     .then(() => {
       // Inserts seed entries
-      return knex('game_join_companies').insert(generateGameJoinCompanies());
+      return knex.raw("COPY game_join_companies (id_game,id_developer,id_publisher,id_platform) FROM '/home/andyoe/Desktop/gameDescription/db/seeds/JoinTable_test.csv' WITH (FORMAT CSV, DELIMITER ',', HEADER)")
```

Comment:
Style thing, but...any reason why this one wasn't kept as a variable? It seems a little easier to read and abstract that way.

Reply:
generateGameJoinCompnies is a function that returns an array of data. knex.raw reads the data directly from the csv file.